pkg/virt-controller/services/template.go

```diff
@@ -527,8 +527,6 @@ func (t *templateService) renderLaunchManifest(vmi *v1.VirtualMachineInstance, i
 			"--container-disk-dir", t.containerDiskDir,
 			"--grace-period-seconds", strconv.Itoa(int(gracePeriodSeconds)),
 			"--hook-sidecars", strconv.Itoa(len(requestedHookSidecarList)),
-			"--less-pvc-space-toleration", strconv.Itoa(t.clusterConfig.GetLessPVCSpaceToleration()),
-			"--minimum-pvc-reserve-bytes", strconv.FormatUint(t.clusterConfig.GetMinimumReservePVCBytes(), 10),
```

Comment:
please squash this into the correct commit, https://github.com/kubevirt/kubevirt/pull/6118/commits/8bbaad210ed5c3aee0030bcb89e61783cace6d02

Reply:
done